src/main/groovy/com/palantir/gradle/javadist/DistributionExtension.groovy

```diff
@@ -19,7 +19,8 @@ class DistributionExtension {
 
     private static final List<String> requiredJvmOpts = [
         '-Djava.security.egd=file:/dev/./urandom',
-        '-Djava.io.tmpdir=var/data/tmp'
+        '-Djava.io.tmpdir=var/data/tmp',
+        '-XX:+PerfDisableSharedMem'
```

Comment:
Probably worth noting the side effect of this is that it will prevent some observability tooling from connecting to the JVM by default (e.g. the java process won't show up in `jps` & `jcmd`, and connecting to it via VisualVM is a bit more difficult). See http://www.evanjones.ca/jvm-mmap-pause.html as well as http://mail.openjdk.java.net/pipermail/hotspot-runtime-dev/2015-March/014324.html and http://bugs.java.com/view_bug.do?bug_id=6447182